G1ANT.Addon.Watson/G1ANT.Addon.Watson.Tests/Commands/watson.classifyimage/ShouldClassifyImage_WhenImageIsClassifiable.G1ANT

```diff
@@ -0,0 +1,25 @@
+addon watson version 2.205.17304.1000
+
+try errorcall DeleteImageAndFailTest
+    call DownloadImage
+    call ClassifyImage
+    call DeleteImage
+    test ⊂♥result == "Valencia orange:0,685;orange:0,821;citrus:0,934;fruit:0,935;navel orange:0,613;mandarin:0,611;clementine:0,541;tangerine:0,5;food:0,798;beverage:0,783;tree:0,798;orange color:1;"⊃
```

Comment:
It would be better if you would check the string on containing some phrases. AI is evolving every day and the values may change.